src/schema/objects/suffixes.yaml

```diff
@@ -42,11 +42,23 @@ DIC:
   display_name: Differential interference contrast microscopy
   description: |
     Differential interference contrast microscopy imaging data
+colFA:
+  value: colFA
+  display_name: Colored Fractional Anisotropy image
+  description: |
+    Diffusion images reflecting the directionality of the diffusion tensor color-coded in red for
+    left-right oriented fibers, in blue for superior-inferior oriented fibers and green for
+    anterior-posterior oriented fibers.
```

Comment:
Sorry to chime in here. I may have missed other related discussions, but I'd dare to say that these are usually referred to as DEC (diffusion-encoded colormaps). Most commonly they contain FA information, but could be other information. They are referred to as `DECFA` in BEP016.

Reply:
I don't think BEP016 ever uses "DECFA" specifically. The fact that the nature of orientation encoding across volumes is directionally-encoded colour, and that the quantitative parameter encoded in the image is FA, are treated very deliberately as two separate dimensions. The distinction between scanner-generated derivatives and pipeline-generated BIDS Derivatives nevertheless means that there doesn't actually need to be correspondence between the two. Indeed a colour FA image generated by a pipeline will *not* use this suffix, even though the information encoded is identical.

I maybe have a slight preference for "colFA" just because it's consistent with data being emitted by many scanners that are intended to be encoded in this way.

Reply:
That BEP016 Google doc is highly outdated; development work on that project transitioned to GitHub some time ago. Maybe that should be flagged near the top of the document, and the website link should be changed. Here's the current status:
https://github.com/bids-standard/bids-bep016/pull/24

Reply:
> Maybe that should be flagged near the top of the document, and the website link should be changed

Done!

Reply:
:+1: thanks for doing this.